test/dotnet-new3.UnitTests/AllProjectsWork.cs

```diff
@@ -1,78 +1,75 @@
-﻿//using System;
-//using System.IO;
-//using System.Linq;
-//using System.Reflection;
-//using System.Runtime.CompilerServices;
-//using Microsoft.DotNet.Cli.Utils;
-//using Microsoft.TemplateEngine.Edge;
-//using Microsoft.TemplateEngine.Mocks;
-//using Microsoft.TemplateEngine.TestHelper;
-//using Microsoft.TemplateEngine.Utils;
-//using Xunit;
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Microsoft.TemplateEngine.Edge;
+using Microsoft.TemplateEngine.Mocks;
+using Microsoft.TemplateEngine.TestHelper;
+using Microsoft.TemplateEngine.Utils;
+using Xunit;
+using Xunit.Abstractions;
 
-//namespace DotnetNew3.UnitTests
-//{
-//    public class AllProjectsWork
-//    {
-//        private static readonly Guid _tempDirParent = Guid.NewGuid();
+namespace dotnet_new3.UnitTests
```

Comment:
Is this according to our namespace naming convention? I have not seen things like this in .net code base - so far.

Reply:
This is how its name in main program, so we are sticking to it for now... https://github.com/dotnet/templating/blob/master/src/dotnet-new3/Program.cs#L19